.travis.yml

```diff
@@ -1,26 +1,29 @@
 language: python
-python:
-  - "2.6"
-  - "2.7"
+python: "2.7"
 sudo: required
+dist: trusty
 services:
   - docker
 env:
   global:
     - PA_TEST_ONLINE_INSTALLER=true
     - PYTHONPATH=$PYTHONPATH:$(pwd)
     - LONG_PRODUCT_TESTS="tests/product/test_server_install.py tests/product/test_status.py tests/product/test_collect.py tests/product/test_connectors.py tests/product/test_control.py tests/product/test_server_uninstall.py"
-  matrix:
-    - OTHER_TESTS=true
-    - SHORT_PRODUCT_TEST_GROUP=0
-    - LONG_PRODUCT_TEST_GROUP_PRESTO_ADMIN="tests/product/test_server_install.py"
-    - LONG_PRODUCT_TEST_GROUP_PRESTO_ADMIN_AND_PRESTO="tests/product/test_status.py"
-    - LONG_PRODUCT_TEST_GROUP_PRESTO_ADMIN_AND_PRESTO="tests/product/test_collect.py tests/product/test_server_uninstall.py"
-    - LONG_PRODUCT_TEST_GROUP_PRESTO_ADMIN_AND_PRESTO="tests/product/test_connectors.py"
-    - LONG_PRODUCT_TEST_GROUP_PRESTO_ADMIN_AND_PRESTO="tests/product/test_control.py"
+
```

Comment:
I think changes in lines 12:22 can be reverted - it's equivalent to the old version, only a bit longer